toolkit/activity.py

```diff
@@ -315,21 +315,21 @@ def _list_tubes_reply_cb(self, tubes):
     def _list_tubes_error_cb(self, e):
         logging.error('ListTubes() failed: %s', e)
 
-    def _new_tube_cb(self, id, initiator, type, service, params, state):
+    def _new_tube_cb(self, tube_id, initiator, type, service, params, state):
```

Comment:
`type` should also change, but I didn't take the time to dig into the callback semantics to determine what the type is of.

Reply:
It is likely that it refers to the type of the Telepathy tube being created, and it is probably an enum defined in the TelepathyGLib library, so it can be changed to `tube_type`